sdk/webpubsub/azure-messaging-webpubsubclient/azure/messaging/webpubsubclient/_client.py

```diff
@@ -910,7 +910,7 @@ def on_close(
                 self._ack_map.clear()
 
                 if self._is_stopping:
-                    _LOGGER.warning("The client is stopping state. Stop recovery.")
+                    _LOGGER.info("The client is stopping state. Stop recovery.")
```

Comment:
When close client normally, there shall be no warning info to confuse SDK users.